test-requirements.txt

```diff
@@ -1,6 +1,5 @@
 locomotivecms @ git+https://github.com/akretion/locomotivecms-python-client.git
 # test requirements
-algoliasearch<2.0.0
```

Comment:
IDT this is going to work w/out https://github.com/shopinvader/odoo-shopinvader/pull/854 but I had no time to test it.

Reply:
It correctly installed algoliasearch<2 because the version specifier is declared upstream. BTW, one can inspect what is installed in the log at the end of the `oca_install_addons` step:

![image](https://user-images.githubusercontent.com/692075/110291641-6e089e00-7fec-11eb-864e-d1b5f3b39226.png)

The important thing in this commit is the removal of the after script that deletes some algolia indexes, which I'm not sure is sill necessary. It should not be necessary because we now use vcr ?

Reply:
Actually it does nothing on travis:

https://travis-ci.com/github/shopinvader/odoo-shopinvader/jobs/488918801#L3125-L3130